juniper_official/lldp/lldp-session-state.rule

```diff
@@ -0,0 +1,173 @@
+/*
+ * Detects LLDP (session) state changes and notifies when anomalies are found.
+ */
```

Comment:
Please maintain separate rule files from maintainability perspective

Reply:
Seperated both the rules